src/data/data.yml

```
@@ -1,2 +1,2 @@
 title: p5.js
-version: 1.1.2
+version: 1.1.8
```

Shouldn't this be 1.1.9? 
```suggestion
version: 1.1.9
```

There might be some stray 1.1.8 lying around the repo, maybe a new release will bring everything up to date or a manual fix would suffice for now.